app/controllers/general.php

```diff
@@ -528,10 +528,10 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
                     $mainDomain = $envDomain;
                 } else {
                     // TODO: @christyjacob remove once we migrate the rules in 1.7.x
-                    if (version_compare(APP_VERSION_STABLE, '1.7.0', '<')) {
-                        $domainDocument = $dbForConsole->findOne('rules', [Query::orderAsc('$id')]);
-                    } else {
+                    if (System::getEnv('_APP_RULES_FORMAT', null) === 'md5') {
```

Comment:
do we need `null` as default value here? it's already `null` by default right?

Reply:
Got you. Yeap, will revert it